js/dataframe/Index.svelte

```diff
@@ -103,7 +103,8 @@
 		{styling}
 		headers={_headers}
 		on:change={(e) => {
-			value = e.detail;
```

Comment:
The problem was that previously Table.svelte was emitting changes upwards but with an empty metadata key, which was overriding the styling. Now we only look at the headers and data that are passed upwards